change/@fluentui-react-charting-20c8ae70-f2a4-4ef4-89b6-7ca932f7481d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28414&commitId=b88622edc573d60956a328aca2935bf428dfe839&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Personas - RTL.default.chromium.png|0| Added|
|ActivityItem.Personas.default.chromium.png|0| Added|
|ActivityItem.Root - RTL.default.chromium.png|0| Added|
|ActivityItem.Root.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
old